bigtable/tests/unit/test_row.py

```diff
@@ -16,6 +16,26 @@
 import unittest
 
 
+class TestRow(unittest.TestCase):
```

Comment:
Thanks for adding this. Slap on the wrist (to me) that `TestRow` didn't already exist.